examples/manipulating_visualizing/plot_prob_atlas.py

```diff
@@ -0,0 +1,60 @@
+"""
+Visualizing 4D atlas maps onto the anatomical image
+===========================================================
+
+Basically this example gives an idea on how to visualize the atlas maps which are
```

Comment:
Remove "Basically". Also "gives an idea". I would say "This example shows how to visualize probabilistic atlases made of 4D images. There are 3 different display types: 1...."